core/src/main/res/layout/reader_fragment_content.xml

```diff
@@ -95,10 +95,10 @@
     android:text="@string/tts_pause"
     android:textStyle="bold"
     android:visibility="gone"
-    app:layout_constraintBottom_toTopOf="parent"
     app:layout_constraintEnd_toStartOf="@id/activity_main_button_stop_tts"
     app:layout_constraintHorizontal_chainStyle="packed"
-    app:layout_constraintStart_toStartOf="parent" />
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent" />
```

Comment:
the pause and stop buttons used to be at the bottom of the screen, it is weird how they are now obscuring the toolbar at the top

Reply:
Should I move to the bottom?

Reply:
yes